index.html

```diff
@@ -39,77 +39,312 @@
+              <br />
+              <h3>About me</h3>
               <br />
               <p>
-                My name is Dagmawi Yohannes I'm a Front End Engineer based in
-                Addis Ababa, Ethiopia ☀️. I describe myself as a passionate
-                developer who loves coding, open source, and the web platform
+                I can help you build a product,
+                feature or website Look through
+                some of my work and experience! If 
+                you like what you see and have a 
+                project to be coded,
+                don't hesitate to Contact me.
               </p>
               <br />
-              <div class="h2"></div>
-              <div class="social-links">
-                <ul class="ulist">
-                  <li class="ulist-icon">
-                    <a href="https://twitter.com/dag_mawi111 "
-                      ><img src="./images/twitter.svg" alt="twitter icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://linkedin.com/in/dagmawi-yohannes-b36488236"
-                      ><img src="./images/Linkedinicon.svg" alt="linkedin icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://medium.com/dagmawi.yoha03"
-                      ><img src="./images/medium.svg" alt="medium icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://github.com/dagmawiii03"
-                      ><img src="./images/github.svg" alt="github icon"
-                    /></a>
-                  </li>
-                </ul>
-              </div>
+              <button>Get My Resume</button>
             </section>
-            <section class="pointer">
+          <section class="abt-card">
+            <section class="card">
+              <br />
+              <img src="./images/languages-icon.png" alt="" />
+              <br />
+              <h1>Languages</h1>
+              <br />
               <ul>
-                <li>
-                  <a href=""><i class="fas fa-chevron-down white-color"></i></a>
-                </li>
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>CSS</li>
               </ul>
             </section>
-          </div>
+          <section class="card">
+            <br />
+            <img src="./images/frameworks-icon.png" alt="" />
+            <br />
+            <h1>Frameworks</h1>
+            <br />
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+            </ul>
+          </section>
+          <section class="card">
+            <br />
+            <img src="./images/skills-icon.png" alt="" />
+            <br />
+            <h1>Skills</h1>
+            <br />
```

Comment:
- [ ] Since you have already used `h1` for `Hi, I'm Dagmawi` please consider using other heading tags like `h2`, `h3`, `h4`  .... :pray:

Reply:
Done